src/frontend/src/pages/CreateAccountPage.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="home">
+    <img
+      width="300px"
```

Comment:
You can move width and height to css by making this image have a class name. Also, I believe the login page has the same size logo so just use those variables from style for reuse!